leakcanary-android-core/src/main/res/values/leak_canary_strings.xml

```diff
@@ -58,6 +58,25 @@
   <string name="leak_canary_stackoverflow_share">Share Leak On Stack Overflow</string>
   <string name="leak_canary_leak_copied">Leak info copied to clipboard</string>
   <string name="leak_canary_leak_clipdata_label">LeakCanary Leak Trace</string>
+  <string name="leak_canary_options_menu_generate_hq_bitmap">Generate HQ Bitmap</string>
+  <string name="leak_canary_options_menu_permission_toast">Please grant the External Storage Permission first, see notification…</string>
+  <string name="leak_canary_generating_hq_bitmap_toast_notice">Rendering HQ Bitmap, this may take a while…</string>
+  <string name="leak_canary_generating_hq_bitmap_toast_failure_notice">Rendering HQ Bitmap, this may take a while…</string>
+  <string name="leak_canary_heap_analysis_list_exception_summary">%d. %s</string>
+  <plurals name="leak_canary_heap_analysis_list_retained_instances">
+    <item quantity="one">%d. Retained Instance</item>
```

Comment:
is that `.` on purpose?

Reply:
Pretty sure; original string interpolation was...

```
titleView.text = "$index. " + (projection.exceptionSummary
// TODO Handle singular
    ?: "${projection.retainedInstanceCount} retained instances")
```

Reply:
This code might be a bit confusing ;) .

All items in the list start with `$index. `, and then some are followed by "1 retained instance", some by "2 retained instances", and some by "RuntimeException Something bad happened".

E.g. :

3. 1 retained instance
2. RuntimeException Something bad happened
1. 4 retained instances

So this needs to change a bit :)

Reply:
Ah yes I was misreading the code a bit; thanks for the example. There was an issue in my Elvis Operator but that's corrected now so the conditional logic now correctly either utilizes `leak_canary_heap_analysis_list_exception_summary` or the quantity specific `leak_canary_heap_analysis_list_retained_instances`. Screenshot below although I wasn't able to find a way to trigger a `RuntimeException Something bad happened` example.

![image](https://user-images.githubusercontent.com/1205783/56928235-077ad680-6a8b-11e9-84ce-4e8f0f8e8922.png)

I also scanned the Sample App to make sure everything is formatting correctly and looks to be so but you might know about corner cases I don't so let me know if there's a specific example you'd like to me double check.

Reply:
Sorry, still not the original behavior :) . There were two numbers in there: first the index in the list. It's the number with a doc, ie 1. 2. 3.  => there shouldn't be two rows with the same number. Then, for retained instances, we show the number of retained instances.

Reply:
Gahh my mistake; should be fixed up now with the latest push. Here's a screenshot...

![image](https://user-images.githubusercontent.com/1205783/56998598-a9202780-6b60-11e9-84ae-d64a6f2511f3.png)

Reply:
🙏